src/main/java/mate/academy/service/ShoppingCartService.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.service;
+
+import mate.academy.exception.ShoppingCartException;
+import mate.academy.model.MovieSession;
+import mate.academy.model.ShoppingCart;
+import mate.academy.model.User;
+
+public interface ShoppingCartService {
+    void addSession(MovieSession movieSession, User user);
+
+    ShoppingCart getByUser(User user) throws ShoppingCartException;
```

Comment:
can be removed after prev comment fix
```suggestion
    ShoppingCart getByUser(User user);
```